src/App.css

```diff
@@ -2,4 +2,232 @@
 *::after,
 *::before {
   box-sizing: border-box;
+  margin: 0;
+}
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,400;1,500&family=Roboto:wght@300;400&display=swap');
+
+.navBar {
+  width: 90%;
+  height: 80px;
+  padding: 0 5%;
+  background-color: #fff;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border-bottom: 1px #e8e8e8 solid;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
- [ ] the line border for the navigation should contain all the width as on the zeplin design. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/71874425/174153190-88aa0ff1-8594-4301-b1b5-d0cf4bf11cc8.png)
 - [ ] The `books-container` has a different background-color than a single book container. take a look at the Zeplin design. I'll suggest you use this color `background-color: #fafafa;` for this background. implement this change on the `books-container` class selector, you can see a red arrow on the part I'm talking about.